user-guide/Advanced_Modules/User Defined APIs/UD_APIs_Viewing_in_Cube.md

```diff
@@ -0,0 +1,18 @@
+---
+uid: UD_APIs_Viewing_in_Cube
+---
+
+# Viewing APIs and tokens in DataMiner Cube
+
+> [!WARNING]
+> This feature is in preview and is not fully released yet. For now, it should only be used on a staging platform. It should not be used in a production environment.
+
+From DataMiner 10.4.0/10.3.5 onwards, you can view an overview of your APIs and tokens in DataMiner Cube.
```

Comment:
```suggestion
From DataMiner 10.4.0/10.3.6 onwards, you can view an overview of your APIs and tokens in DataMiner Cube.
```
Just confirmed with Matthias, there is a check that won't show the UI if the DMA is not in the Skyline internal network. So the cube feature (with soft-launch flag) will only be available in 10.3.6